cubes/sql/query.py

```diff
@@ -957,7 +957,7 @@ def conditions_for_cuts(self, cuts):
         conditions = []
 
         for cut in cuts:
-            hierarchy = str(cut.hierarchy) if cut.hierarchy else None
+            hierarchy = str(cut.hierarchy) if cut.hierarchy else "default"
```

Comment:
Guys, we need this fix in master.

And the few lines below, in case of SetCut, we need to use "hierarchy" var instead of "str(cut.hierarchy)".
Here: https://github.com/DataBrewery/cubes/blob/master/cubes/sql/query.py#L973